src/components/common/NameRequestInfo.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div id="name-request-summary">
+    <v-row id="name-request-info">
+      <v-col>
+        <label>
+          <strong>Name Request</strong>
+        </label>
+      </v-col>
+      <v-col>
+        <ul>
+          <li class="name-request-title mb-4">
+            <strong>{{ getBusinessIdentifier }}</strong> - {{ getNameRequestDetails.approvedName}}
+          </li>
+          <li>Entity Type: {{ entityTypeDescription() }}</li>
+          <li class="mb-4"> Request Type: {{ requestType() }}</li>
+          <li>Expiry Date: {{ formattedExpirationDate() }}</li>
+          <li>Status: {{ getNameRequestDetails.status }}</li>
+          <li>Condition/Consent: {{ conditionConsent() }}</li>
+        </ul>
+      </v-col>
+    </v-row>
+    <v-row id="name-request-applicant-info">
+      <v-col>
+        <label>
+          <strong>Name Request Applicant</strong>
+        </label>
+      </v-col>
+      <v-col>
+        <ul>
+          <li>Name: {{ applicantName() }}</li>
+          <li>Address: {{ applicantAddress() }}</li>
+          <li>Email: {{ getNameRequestApplicant.emailAddress }}</li>
+          <li>Phone: {{ getNameRequestApplicant.phoneNumber }}</li>
+        </ul>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Mixins, Vue } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { getName } from 'country-list'
+
+// Enums
+import { NameRequestStates } from '@/enums'
+
+// Interfaces
+import { GetterIF, NameRequestDetailsIF, NameRequestApplicantIF } from '@/interfaces'
+
+// Mixins
+import { DateMixin } from '@/mixins'
+
+@Component
+export default class NameRequestInfo extends Mixins(DateMixin) {
+  // Global getters
+  @Getter isEntityType!: GetterIF;
+  @Getter isTypeBcomp!: GetterIF;
+  @Getter isTypeCoop!: GetterIF;
+  @Getter getBusinessIdentifier!: GetterIF;
+  @Getter getNameRequestDetails!: NameRequestDetailsIF;
+  @Getter getNameRequestApplicant!: NameRequestApplicantIF;
+  getName = getName
+
+  /** The entity title  */
+  entityTypeDescription () :string {
+    if (this.isTypeBcomp) {
+      return 'BC Benefit Company'
+    } else if (this.isTypeCoop) {
+      return 'BC Cooperative Association'
+    }
+
+    return ''
+  }
+
+  /** The request type */
+  requestType () : string {
```

Comment:
FYI, these could be getters (but I'm not saying to change it), and they should be private.

Reply:
Updated to private